data/events/2024/portugal/main.yml

```
@@ -0,0 +1,417 @@
+name: "2024-portugal" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "Portugal" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayspt" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "DevOpsDays Portugal is back! 2024 is the third edition of the event, and it's powered by DevOps Porto, one of the communities dedicated to DevOps in Portugal. This year, we repeat the event in the beautiful city of Porto as the host city." # Edit this to suit your preferences
+ga_tracking_id: # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+event_group: "Portugal"
+
+# All dates are in unquoted YYYY-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow YYYY-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-09-23T00:00:00+01:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-09-24T00:00:00+01:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2024-03-11T00:00:00+01:00 # start accepting talk proposals.
+cfp_date_end: 2024-05-05T23:00:00+01:00 # close your call for proposals.
+cfp_date_announce: 2024-05-19T23:00:00+01:00 # inform proposers of status
+
+cfp_open: "false"
+cfp_link: #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+cancel: "false"
+
+# Location
+#
+coordinates: # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: # Defaults to city, but you can make it the venue name.
+#
+location_address: #Optional - use the street address of your venue. This will show up on the welcome page if set.
```

You appear to have details in `location.md`, so please reflect them here (assuming `location.md` is accurate). For example:

```suggestion
location: # Defaults to city, but you can make it the venue name.
#
location_address: "EDIFÍCIO DA ALFÂNDEGA, R. Nova da Alfândega, 4050-430 Porto"
```

Fwiw, this feels like you copied a `main.yml` from a previous year, because the template has changed; for example, `coordinates:` isn't used anymore…